pkg/target/manager.go

```diff
@@ -473,6 +487,10 @@ func (m *managerImpl) getTarget(t Target) (Target, error) {
 	return t, err
 }
 
-func (m *managerImpl) GardenClient(name string) (gardenclient.Client, error) {
-	return GardenClient(name, m.config, m.clientProvider)
+func (m *managerImpl) GardenClient(gardenIdentity string) (gardenclient.Client, error) {
+	return GardenClient(gardenIdentity, m.config, m.clientProvider)
+}
+
+func (m *managerImpl) GardenClientForKubeconfig(kubeconfigFile string, contextName string) (gardenclient.Client, error) {
+	return GardenClientForKubeConfig(kubeconfigFile, contextName, m.clientProvider)
```

Comment:
I'm not sure if these two functions `GardenClient` and `GardenClientForKubeconfig` are in the right place here (in the manager).

Reply:
I agree. Also the ClientProvider has nothing to do with the `target.Manager`

Reply:
I have not found any usage for this method. We do we call this on the manager?